packages/data/src/macros/macros_mm.yml

```diff
@@ -144,6 +144,7 @@
 "can_pass_gibdo": "has(MASK_GIBDO) && soul_redead_gibdo"
 "can_get_gossip_fairy": "can_play_healing || can_play_epona"
 "can_jumpslash": "has_weapon || has_sticks || has_mask_zora"
+"can_enter_evans_before_gb_clear": "has_mask_zora"
```

Comment:
Could you add the `short_hook_anywhere && trick(MM_ZORA_HALL_DOORS)` to this? Would need to go to core/lib/combo/settings/tricks.ts to add it. Description could be "Access the Doors in Zora Hall using Short Hookshot Anywhere" and then would also need added in core/lib/combo/preset.ts to the hell preset since it uses every trick.

I did confirm it works on all four doors